### DISCUSSION TITLE
F41254A3-5F26-4DF7-840D-5E83C6D8649B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#178] [General]
# MLP planet

Mlp planet Mlp g4

https://celestia.mobi/addon?item=F41254A3-5F26-4DF7-840D-5E83C6D8649B

**Comment by FaithfulCocoa** [comment]
this is NOT a MLP planet...

**Comment by exocritic5457** [comment]
Ignoring the glaring fact that this doesn't even work, here's a random tidbit: Equestria would easily count as a Death World by Warhammer 40K standards
